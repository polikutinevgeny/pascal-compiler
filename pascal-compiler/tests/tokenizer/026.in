4.32e--4