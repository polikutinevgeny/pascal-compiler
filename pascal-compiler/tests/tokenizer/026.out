Line |Pos  |Type        |Subtype                  |Source                             |Value                                             
----------------------------------------------------------------------------------------------------------------------------------------------
Unexpected character: '-'(#45) at 1:7
